azure/services/scalesets/scalesets.go

```diff
@@ -646,3 +659,13 @@ func base64EncodedHash(vmss compute.VirtualMachineScaleSet) (string, error) {
 	_, _ = hasher.Write(jsonData)
 	return base64.URLEncoding.EncodeToString(hasher.Sum(nil)), nil
 }
+
+// contains tells whether a contains x.
+func contains(a []string, x string) bool {
```

Comment:
consider moving this helper to the common util package

Reply:
Done.